src/main/java/org/apache/commons/lang3/math/IEEE754rUtils.java

```diff
@@ -41,6 +41,7 @@ public static double min(final double... array) {
         Validate.isTrue(array.length != 0, "Array cannot be empty.");
 
         // Finds and returns min
+        @SuppressWarnings("index:array.access.unsafe.high.constant") // Validate.isTrue(array.length != 0, "Array cannot be empty.") => array has a non zero length
```

Comment:
Instead of suppressing this warning, I would annotate the argument to this method to express this fact. This follows the principle of ["annotate normal behavior"](https://checkerframework.org/manual/#annotate-normal-behavior). Everywhere that you've suppressed a warning with a message about a call to `Validate.isTrue`, you can probably add an annotation instead. For example, in this method you'll need an `@MinLen(1)` annotation on the parameter.